automation/submission-perf-test-2.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
```

Comment:
we should rename this to overwrite the existing file so it's easier to see the diffs